src/core/mesh/qgsmeshlayer.cpp

```diff
@@ -312,6 +379,7 @@ void QgsMeshLayer::assignDefaultStyleToDatasetGroup( int groupIndex )
 
 QgsMapLayerRenderer *QgsMeshLayer::createMapRenderer( QgsRenderContext &rendererContext )
 {
+  QgsEventTracing::ScopedEvent tracing( "Mesh renderer", "create map renderer" );
```

Comment:
I would also suggest to remove all the tracing events from the code... so that when someone else tries to trace something else, this does not get in the way...?